frontend/src/board/board.ts

```diff
@@ -45,5 +47,105 @@ export class Board {
       { row: 0, column: 2},
       { row: 0, column: 3},
     ];
+
+  }
+
+  findMoves() {
+    return {
+      north: this.checkDirections(this.robotPositions[0].row, this.robotPositions[0].column, 'North'),
```

Comment:
Indentation looks a little funny here, I think we need to move south, east, west to this indentation level as well